File: studies/src/main/java/io/innofang/studies/StudiesActivity.java
package io.innofang.studies;

import android.support.v4.app.Fragment;

import org.jetbrains.annotations.NotNull;

import cn.edu.nuc.library.base.FragmentContainerActivity;

public class StudiesActivity extends FragmentContainerActivity {

    @NotNull
    @Override
    protected Fragment createFragment() {
        return StudiesFragment.newInstance();
    }

    @Override
    public int getLayoutResId() {
        return R.layout.sd_activity_studies;
    }

    @Override
    public int getFragmentContainerId() {
        return R.id.sd_fragment_container;
    }
}

File: studies/src/main/java/cn/edu/nuc/studies/StudiesActivity.kt
package cn.edu.nuc.studies

import android.support.v4.app.Fragment
import cn.edu.nuc.library.base.FragmentContainerActivity


class StudiesActivity : FragmentContainerActivity() {

    override val layoutResId: Int = R.layout.sd_activity_studies

    override val fragmentContainerId: Int = R.id.sd_fragment_container

    override fun createFragment(): Fragment = StudiesFragment.newInstance()

}
